script.js

```diff
@@ -315,11 +291,5 @@ for (let i = 0; i < collection.length; i++) {
   
```

Comment:
Too much empty space used on each method and functions, remove them if not used, you can give it one line space, if you want to organize it better and to point to specific stuff in your code, but there is no point of using 3-4 empty lines of code, also keep the same method of writing. You're using everywhere semicolon except on line 289 check that stuff on other places too.

Reply:
done